src/Resources/contao/modules/ModuleArticle.php

```diff
@@ -366,7 +366,7 @@ public function generatePdf()
 		// Close and output PDF document
 		$pdf->lastPage();
 		$pdf->Output(standardize(ampersand($this->title, false)) . '.pdf', 'D');
-
+		// FIXME: This has to be changed to a ResponseException somehow - rather write to temp file and stream it then?
```

Comment:
This is btw. the reason why I wanted to steal the implementation from https://github.com/contao/core-bundle/pull/68#discussion_r27342929 into my PR.